paper.tex

```diff
@@ -136,6 +136,34 @@ \subsection{Refunding}
 
 Where $price$  is the subscription price, $M$ is the number of expected data samples, $F_{b}$ is the brokerage fee(\%), $F_{t}$ is the transaction fee of the smart contract(\%), $N$ is the number of subscribers in this contract.
 
+\subsection{Game Theory Evaluation}
```

Comment:
It is the first time where Game Theory takes place. Before that, you shall mention in introduction and even related work sections.